.claude/hooks/track.sh

```
@@ -2,56 +2,116 @@
 
 set -euo pipefail
 
 input=$(cat)
 prompt=$(printf '%s' "$input" | jq -r '.prompt // empty' 2>/dev/null || printf '')
 
-# `$track ...` (whitespace required after, so `$tracking`/`$tracked` don't match)
-# anywhere, or a line starting with `track: ...`.
+legacy_note=''
+
+emit_list() {
+  printf 'TASK-PLANNING PROJECT DIRECTIVE: `$project list` — args: "%s". Operate on .agents/projects/registry.yml per the task-planning skill. Render the active projects as a markdown table whose FIRST column is a 1-based row number (# | name | status | user | host | one-line summary). BY DEFAULT show only projects whose `user` matches the current user (run `whoami`); if none match, say so and show all. Args `all`: list every user. Then tell the user they can reply with a row number to resume that project — a lone number in their next message means "resume the project at that row" (same as `$project resume <name>`). Confirm in one short line.\n' "$1"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)